### PR TITLE
[Bugfix#1931] add a popup when the user wants to remove the social link from the profile

### PR DESCRIPTION
**Pull request by JimHolman1199**
add a popup when the user wants to remove the social link from the profile
fix test

### CODE REVIEW

**Review by luchward — APPROVED** [review]

File: src/app/component/user/components/profile/edit-profile/social-networks/social-networks.component.spec.ts [hunk 1]
```diff
@@ -22,6 +40,8 @@ describe('SocialNetworksComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(SocialNetworksComponent);
     component = fixture.componentInstance;
+    dialog = TestBed.get(MatDialog);
+
```
Comment (skrebetsnataliia) [inline src/app/component/user/components/profile/edit-profile/social-networks/social-networks.component.spec.ts:44]: Why did you need empty line?

**Review by skrebetsnataliia — APPROVED** [review]
